specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/cluster.json

```diff
@@ -0,0 +1,1154 @@
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}": {
+            "put": {
+                "tags": [
+                    "Clusters"
+                ],
+                "operationId": "Clusters_Create",
+                "description": "Creates a new HDInsight cluster with the specified parameters.",
+                "x-ms-examples": {
+                    "Create Hadoop on Linux cluster with SSH password": {
+                        "$ref": "./examples/CreateLinuxHadoopSshPassword.json"
+                    },
+                    "Create Hadoop on Linux cluster with SSH public key": {
+                        "$ref": "./examples/CreateLinuxHadoopSshPublicKey.json"
+                    },
+                    "Create Spark on Linux Cluster with SSH password": {
+                        "$ref": "./examples/CreateLinuxSparkSshPassword.json"
+                    },
+                    "Create Secure Hadoop cluster": {
+                        "$ref": "./examples/CreateLinuxHadoopSecureHadoop.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ClusterCreateParametersExtended"
+                        },
+                        "description": "The cluster create request."
+                    }
+                ],
+                "responses": {
+                    "200": {

```

Comment:
>200 [](start = 21, length = 3)

Does this operation support 202 response?

Reply:
Currently, only 200 is expected.